### ISSUE TITLE
About pretrained-dataset

### ISSUE THREAD

**Issue by zzllc3** [xupine/DFPENet#1]
Thank you for your remarkable work. In your paper, you said that using 4 images from validation set. Dose the validation here means the test set(17 images)?
